### ISSUE TITLE
VSC Should Be Able To Also Print From The Menu 

### ISSUE THREAD

**Issue by Neohiro79** [PeterWone/vsc-print#99]
I don't think that's worth the time, just to figure out how to put a single `text-string` into a menu you need to clone a "gigabyte" project. I mean what a waste of resources.

---

> What I'd like to see is support for adding things to the main menu, so I can put a Print item where it belongs.

---

I was looking a little bit into the electron documentation, since it's javascript-based:
https://www.electronjs.org/docs/api/menu

**Maybe try this:**
Menu.getApplicationMenu();

**To append "something":**
menu.append(menuItem);

**Also:**
https://www.electronjs.org/docs/api/menu-item

---

**Additionally, you might want to look into this example, since it's Javascript, this should work:**
https://codepen.io/neohiro79/pen/xxgzywm?editors=1111

Since I haven't worked with electron yet, I cannot tell if this works - but since it's Javascript, it should.

---

**NOTE:**
Basically it's `"grabbing the CONTENT of the object"` (in complete), `"just change the whole object as you wish"`, **then just push it back**. Since it's Javascript, there is no such thing like "protection" of "methods" or even functions and objects or whatever.

But if they have been intelligent, they included all extension-includes "above" the last menu-object-var in the program-flow.
Since the last include counts, my example won't work as expected.

**Comment by PeterWone** [comment]
I'm going to close this. It's not something we can fix. There _is_ some interest from the vs code team in supporting configurable submenus off the main menu but don't hold your breath. If they ever do support it you can bet your booties I will put "Print" where it belongs in the File menu.
